### PR TITLE
Knowledge graph

### PR DESCRIPTION
**Pull request by jetstream131**
Risk factor client

### CODE REVIEW

**Comment by dquig** [comment]
We should have an interface, something like FuzzySearchable, that all the clients implement. with the closestMatches method

**Comment by dquig** [comment]
We need to add the risk factors to the hard coded conditions as well

**Comment by dquig** [comment]
We need to add the risk factors to the hard coded conditions as well

File: query_proj/main.py [hunk 1]
```diff
@@ -174,7 +174,7 @@ def getSympAav(cond):
 
 def produces(a):
     for i in a:
-        str = ".add(PRODUCES_SYMPTOM_IRI,\"{}\")".format(i)
```
Comment (dquig) [inline query_proj/main.py]: Copy and paste problems?

Comment (jetstream131) [inline query_proj/main.py]: didn't want to manually type everything haha

**Pull request merged**